prediction_market_agent_tooling/markets/omen/omen.py

```diff
@@ -149,6 +149,51 @@ def get_liquidity(self) -> TokenAmount:
     def get_tiny_bet_amount(cls) -> BetAmount:
         return BetAmount(amount=0.00001, currency=cls.currency)
 
+    @staticmethod
+    def get_bet_outcome(binary_outcome: bool) -> str:
+        return OMEN_TRUE_OUTCOME if binary_outcome else OMEN_FALSE_OUTCOME
+
+    @staticmethod
+    def get_bet_outcome_bool(bet_outcome: OutcomeStr) -> bool:
+        return True if bet_outcome == OMEN_TRUE_OUTCOME else False
+
+    def liquidate_existing_positions(
+        self, bet_outcome: bool, web3: Web3 | None = None
+    ) -> float:
+        """
+        Liquidates all previously existing positions.
+        Returns the amount in collateral obtained by selling the positions.
+        """
+        better_address = APIKeys().public_key
```

Comment:
```suggestion
        better_address = APIKeys().bet_from_address
```

I guess this is preferred in this function? (in case that bets are from Safe)